### PR TITLE
[562503] OAIB Insert Remove Operational Activities wrongly filters activities allocated to actors

### PR DESCRIPTION
**Pull request by pdulth**
Bug: 562503
Change-Id: If672108c2801c3f8f6a6a9d6b368c017a37de239
Signed-off-by: Philippe DUL <philippe.dul@thalesgroup.com>

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.sirius.analysis/description/oa.odesign [hunk 1]
```diff
@@ -418,14 +418,14 @@
...
             <element name="element"/>
             <elementView name="elementView">
-              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctions" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctions()->select(getDisplayedAbstractFunction | getDisplayedAbstractFunction.allocationBlocks->filter(cs::Component)->select(c | c.isActor())->size() = 0))"/>
-              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctionViews" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctionViews()->select(getDisplayedAbstractFunctionView | getDisplayedAbstractFunctionView.target.allocationBlocks->filter(cs::Component)->select(c | c.isActor())->size() = 0))"/>
+              <subVariables xsi:type="tool:AcceleoVariable" name="visibleFunctions" computationExpression="aql:self.void2Null(elementView.getDisplayedAbstractFunctions())"/>
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.sirius.analysis/description/oa.odesign:421]: Its now consistent with SDFB / LDFB and PDFB

**Comment by pdulvp** [comment]
Failing junit shall be fixed by https://github.com/eclipse/capella/pull/212

**Comment by pdulth** [comment]
I just done a rebase, Failing junit shall be fixed by https://github.com/eclipse/capella/pull/212

File: core/plugins/org.polarsys.capella.core.sirius.analysis/description/oa.odesign (hunk 1 above)
Comment (sandupostaru) [reply core/plugins/org.polarsys.capella.core.sirius.analysis/description/oa.odesign:421]: Do we know why we previously chose to filter activities allocated to actors?

**Pull request merged**